apps/cms-v2/package.json

```diff
@@ -14,7 +14,21 @@
   "dependencies": {
     "@datocms/cma-client-browser": "2.0.0",
     "@hookform/resolvers": "^3.3.1",
+    "@opentelemetry/api": "../../node_modules/@opentelemetry/api",
+    "@opentelemetry/api-logs": "../../node_modules/@opentelemetry/api-logs",
+    "@opentelemetry/core": "../../node_modules/@opentelemetry/core",
+    "@opentelemetry/exporter-logs-otlp-http": "../../node_modules/@opentelemetry/exporter-logs-otlp-http",
+    "@opentelemetry/exporter-trace-otlp-http": "../../node_modules/@opentelemetry/exporter-trace-otlp-http",
+    "@opentelemetry/instrumentation-http": "../../node_modules/@opentelemetry/instrumentation-http",
+    "@opentelemetry/instrumentation-winston": "../../node_modules/@opentelemetry/instrumentation-winston",
+    "@opentelemetry/resources": "../../node_modules/@opentelemetry/resources",
+    "@opentelemetry/sdk-logs": "../../node_modules/@opentelemetry/sdk-logs",
+    "@opentelemetry/sdk-node": "../../node_modules/@opentelemetry/sdk-node",
+    "@opentelemetry/sdk-trace-base": "../../node_modules/@opentelemetry/sdk-trace-base",
+    "@opentelemetry/sdk-trace-node": "../../node_modules/@opentelemetry/sdk-trace-node",
+    "@opentelemetry/semantic-conventions": "../../node_modules/@opentelemetry/semantic-conventions",
```

Comment:
question: Why do we need to do it like this?

Reply:
Its monorepo-world. Some packages, including otel, are very heavy relying on globals. 

In this repo, there are 2 packages that must install packages - app and "packages/otel". If we install them "normally" its very likely that packages will be duplicated and don't point on the same instance.

This pattern is to ensure there will be only single instance of OTEL in node_modules, all packages must point to this instance